frontend/src/v5/validation/schemes.ts

```diff
@@ -20,11 +20,93 @@ import { formatMessage } from '@/v5/services/intl';
 import { getPasswordStrength } from '@/v4/services/validation';
```

Comment:
I think this file is turning out too big, can we split it? Also that would follow the idea from the rest of the files in the directory and I already see defined there a schema for password in auth.ts ? can we unify that so we dont repeat code?

Reply:
There is now a new structure which aims at being much cleaner:

validation/
├─ userSchemes/
│  ├─ validators.ts
│   `─ <one file for each related schemes>
 `─ containerAndFederationSchemes/
   ├─ validators.ts
    `─ <one file for each related schemes>